commandline/src/main/java/gov/cms/qpp/conversion/ConversionFileWriterWrapper.java

```diff
@@ -60,14 +60,12 @@ private void executeConverter(Converter converter) {
 		try {
 			JsonWrapper jsonWrapper = converter.transform();
 			Path outFile = getOutputFile(source.getName(), true);
-			DEV_LOG.info("Successful conversion.  Writing out QPP to {}",
-				outFile.toString());
+			DEV_LOG.info("Successful conversion. Writing out QPP to {}", outFile);
 			writeOutQpp(jsonWrapper, outFile);
-		} catch (TransformException exception) {
-			AllErrors allErrors = exception.getDetails();
+		} catch (TransformException ignore) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)